docs/HowTo/Monitor/Logging.md

```diff
@@ -0,0 +1,96 @@
+---
+description: Teku log level settings and log formatting
+---
+
+# Logging
+
+Teku uses Log4J2 for logging, and provides multiple methods to configure logging behavior:
+
+* [Basic](#basic-log-level-setting) - changes the log level.
+* [Destination logging](#configure-log-destination) - configure the destination for log output.
+* [Custom logging](#advanced-custom-logging) - custom logging to configure the output and format
+    of the logs.
+
+## Basic log level settings
+
+Use the [`--logging`](../../Reference/CLI/CLI-Syntax.md#logging) command line option to specify
+logging verbosity. The [`--logging`](../../Reference/CLI/CLI-Syntax.md#logging) option changes the
+volume of events displayed in the log. Valid log levels are `OFF`, `FATAL`, `ERROR`, `WARN`,
+`INFO`, `DEBUG`, `TRACE`, `ALL`. The default level is `INFO`.
+
+By default most logging output is sent to the log file, and limited content shown on the console.
+
+!!! tip
+
+    Use the [`log_level`](https://pegasyseng.github.io/teku/#operation/putAdminLog_level) API
+    method to change the log level while Teku is running.
+
+Additional logging options include:
+
+* [`--log-color-enabled`](../../Reference/CLI/CLI-Syntax.md#log-color-enabled)
+    displays status and event log messages in different colors on the console
+* [`--log-include-events-enabled`](../../Reference/CLI/CLI-Syntax.md#log-include-events-enabled)
+    logs frequent update events. For example every slot event with validators and attestations.
+* [`--log-include-validator-duties-enabled`](../../Reference/CLI/CLI-Syntax.md#log-include-validator-duties-enabled)
+    logs details of validator event duties.
+
+## Configure log destination
+
+Use the [`--log-destination`](../../Reference/CLI/CLI-Syntax.md#log-destination) command line
+option to specify where to output log information. Valid options are `BOTH`, `CONSOLE`,
+`DEFAULT_BOTH`, `FILE`. Defaults to `DEFAULT_BOTH`.
+
+When using `BOTH` or `DEFAULT_BOTH`, system updates such as blockchain events
+are displayed on the console, and errors and other information are logged to a file. The log file
+location can be specified with the [`--log-file`](../../Reference/CLI/CLI-Syntax.md#log-file)
+command-line option.
```

Comment:
Suggest making the last sentence active: 

Specify the log file location with the --log-file command line option